templates/controllervm/cloudinit/networkdata

```diff
@@ -1,4 +1,7 @@
 version: 2
 ethernets:
   eth1:
-    addresses: [ {{ .ControllerIP }} ]
\ No newline at end of file
+    addresses: [ {{ .ControllerIP }} ]
+{{if .Gateway }}
+    {{ .Gateway }}
```

Comment:
this should be either gateway4: {{ .Gateway }} or gateway6: {{ .Gateway }}

https://cloudinit.readthedocs.io/en/latest/topics/network-config-format-v2.html#examples

Reply:
oh wait ignore that, you do that when setting templateParameters["Gateway"]

Reply:
Yeah, I could have implemented it differently, this way seemed cleaner.